Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -102,6 +121,18 @@ class MainTabBarViewController: UITabBarController {
             root: tabRoot, title: "Transactions", image: UIImage(named: "tab-icon-transactions")!, tag: 1)
     }
 
+    #warning("fix tracking")
+    private func dappsTabViewController() -> UIViewController {
+        let noSafesVC = NoSafesViewController()
+        let loadSafeViewController = LoadSafeViewController()
+        loadSafeViewController.trackingEvent = .settingsSafeNoSafe
```

Comment:
the tracking event should be different one

Reply:
fixed